poppy/poppy_core.py

```diff
@@ -2104,15 +2136,20 @@ def getPhasor(self,wave):
         else:
             return self.phasor
 
+    def getPhasor(self,wave):
+        warnings.warn("getPhasor is deprecated; use get_phasor instead", DeprecationWarning)
+        return self.get_phasor(wave)
+
+
     def display(self, nrows=1, row=1, what='intensity', crosshairs=True, ax=None, colorbar=True,
```

Comment:
Right now `opd_vmin` is always set to `-1*opd_vmax` so you get a balanced display scale. I haven't encountered any cases where I needed different positive and negative maxes so I just left it simple like that. Same with leaving the transmission as 0-1 always. 

I agree these are reasonable additions but not part of this PR. Feel free to submit another PR with them if you want! 👍